Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewController.swift

```diff
@@ -323,13 +323,18 @@ class CreditCardBottomSheetViewController: UIViewController, UITableViewDelegate
     }
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        guard let plainTextCreditCard = viewModel.getPlainCreditCardValues(
-            bottomSheetState: .selectSavedCard,
-            row: indexPath.row
-        ) else {
+        guard let hostingCell = tableView.dequeueReusableCell(
+            withIdentifier: HostingTableViewCell<CreditCardItemRow>.cellIdentifier) as? HostingTableViewCell<CreditCardItemRow>,
+              let plainTextCreditCard = viewModel.getPlainCreditCardValues(
+                bottomSheetState: .selectSavedCard,
+                row: indexPath.row
+              )
+        else {
             return
         }
         didSelectCreditCardToFill?(plainTextCreditCard)
+        hostingCell.backgroundColor = themeManager.currentTheme.colors.layer5Hover
```

Comment:
We should use the cell `isHighlighted` or `setHighlighted` for highlight colors, as we do with others buttons/cells